packages/mui-material-next/src/InputBase/InputBase.tsx

```diff
@@ -375,7 +366,7 @@ const InputBase = React.forwardRef(function InputBase<
   };
 
   const handleClick = (event: React.PointerEvent) => {
-    if (onClick /* && !fcs.disabled */) {
+    if (onClick) {
```

Comment:
Why are we no longer checking for the disabled state?

Reply:
@mnajdova It never should have been checking for `!fcs.disabled` here - it was part of https://github.com/mui/material-ui/pull/36892 which was reverted!